test/fixtures/export/Do not caps-protect name fields #384.noopsort.bibtex

```diff
@@ -34,12 +34,12 @@
 }
 
 @book{bovendeert,
-  author = {\noopsort{Eert}{boven d'\relax{} Eert}, Christianus},
+  author = {{\noopsort{eert}}{boven d'Eert}, Christianus},
   note = {[boven d'Eert][Christianus]}
 }
 
 @book{s-gravesande,
-  author = {\noopsort{Gravesande}'s-\relax{} Gravesande, Goverdus},
+  author = {{\noopsort{gravesande}}{'s-Gravesande}, Goverdus},
```

Comment:
So this means that if noopsort is in effect, no `relax` is added, but if a NDP is found, `NDP + family` must always be grouped?